content/docs/gke/troubleshooting-gke.md

```diff
@@ -19,10 +19,18 @@ For more help, try the
 
 Here are some tips for troubleshooting Cloud IAP.
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
+* Make sure you are using HTTPS.
+* Check project [quota page](https://pantheon.corp.google.com/iam-admin/quotas) to see if there are any quota limit, increase them as needed.
```

Comment:
I'm not sure what this bit means: "to see if there are any quota limit". I guess there are always quota limits. Does it mean that the console will display info about exceeded quota?